appholder/src/main/java/com/android/identity/wallet/util/ProvisioningUtil.kt

```diff
@@ -119,9 +120,13 @@ class ProvisioningUtil private constructor(
 
         val pendingCredsCount = DocumentUtil.managedCredentialHelper(
             document,
-            secureArea,
-            settings,
             CREDENTIAL_DOMAIN,
+            {toBeReplaced -> SecureAreaBoundCredential(
```

Comment:
This should be MdocCredential, right? I mean, it works with `SecureAreaBoundCredential` but it is an mdoc so we might as well use that instead. Ditto other places in this app.